ExeterTeam25/group-software-project
# Implemented GDPR into the site

To test:
- open the site
- a banner should show to accept cookies
- accept cookies
- scroll to the bottom
- you should see 1 button and 3 links
- test the button opens the cookie preferences
- click on all three links
- each link should show the correct document (tos, cookie policy, and privacy policy)


Details:
- added consent banner
- added cookie preferences
- added terms of service (tos)
- added cookie policy
- added privacy policy
